packages/ra-data-simple-prisma/src/createHandler.ts

```diff
@@ -151,5 +151,7 @@ export const createHandler = async <Args extends CreateArgs>(
     await auditHandler(req, options?.audit, created);
   }
 
-  res.json({ data: created });
+  const response = { data: created };
+  res.json(response);
+  return response;
```

Comment:
I would probably not return an object with "data" prop, why would you need it? can it be "created" directly?

Reply:
I wanted to return the exact same piece of data that is also sent as a response. Is most cases this is basically the `data` wrapper, but in list responses this also gives you the total number of items, which could be userfull for other implementations as well so they don't need to fetch it anymore.